### ISSUE TITLE
Codedoc runs fine locally but formatting is messed up on Github Pages

### ISSUE THREAD

**Issue by beauwilliams** [CONNECT-platform/codedoc#96]
Hi, I have been using codedoc for a while

Not sure what I am doing wrong today..

Codedoc check is fine, site runs fine locally via codedoc serve

Yet the site when deployed is not right.. https://beauwilliams.github.io/Android-Study-Jams/

![image](https://i.ibb.co/fYPdZrH/Screen-Shot-2021-01-28-at-11-57-53-am.png)

Any clue what might be the issue?

**Comment by beauwilliams** [comment]
It also will 404 when trying to visit a page https://beauwilliams.github.io/docs/session1

**Comment by beauwilliams** [comment]
I have built and deployed the boilerplate and same result. I followed docs to a t. Yet same issue. See here --> https://beauwilliams.github.io/codedoc/

Is it just me?

**Comment by beauwilliams** [comment]
Now, I wanted to see if it was a github issue so I cloned an old repo _that contained a codedoc wiki_ and deployed it. Its working just fine. In must be an issue with codedoc, but not even remotely where to start + actually need to write these docs today. See here --> https://beauwilliams.github.io/DSCAndroidStudyJams/

**Comment by beauwilliams** [comment]
Running `codedoc build` on this old repo produces the same error. Now https://beauwilliams.github.io/DSCAndroidStudyJams/ is broken too

**Comment by loreanvictor** [comment]
ok I am labeling this as a bug for now, because from what you've described its either a confusing behavior from codedoc or confusing information on codedoc documentations, and in either case requires an action.

I personally use codedoc frequently and do not have these issues, though I use a slightly different setup than whats on the docs. I will look into the issue later today, but in the meanwhile, if you are in a rush @beauwilliams, I could recommend looking at [this](https://github.com/loreanvictor/tyfon/), [this](https://github.com/loreanvictor/callbag-jsx/), [this](https://github.com/loreanvictor/callbag-common/) or [this](https://github.com/loreanvictor/themed-jss/) repo (they are all fairly recent and I believe they all use the latest version of codedoc) and using the same codedoc+gh-actions setup for your project.

**Comment by loreanvictor** [comment]
p.s. @beauwilliams if you came to a conclusion on why you are getting these issues from comparison with those repos, I would also greatly appreciate PRs (either here or in [codedoc-docs](https://github.com/CONNECT-platform/codedoc-docs)).

**Comment by beauwilliams** [comment]
I noticed this, in my previous repo's that had a working build we can see the root of the cwd in the path for the href tags

`<a href="/MSA-AUS-2020-FlaskWebsite/docs/Breakdown">Stack Breakdown</a>`

In the builds today we see 

<a href="/docs/session1">Study Jams Session 1</a>

I fixed thi issue by adding the root path in front as seen here, page now load fine [commit](https://github.com/beauwilliams/Android-Study-Jams/commit/c0186ae467b629763b82e4e014694c1c348107c6#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051)

Not sure why this is occurring but will see if I can figure it out later

**Comment by beauwilliams** [comment]
Rolling back cli and core to 0.2.0 did not seem to fix the issue either with fresh init and build

**Comment by loreanvictor** [comment]
that makes me think that perhaps the [project namespace](https://codedoc.cc/docs/config/output#project-namespace) is missing? the CLI should usually add the namespace on its own when working with a github repository, so thats strange.

anyways could you add a namespace and see if this fixes the issue?

**Comment by beauwilliams** [comment]
Darn it.. yeah that was it. 

Look I am really sorry, it's always funny coming back to some code after a decent break. Must have completely forgotten this detail and then missed it in the docs to just for extra good measure. Thank you 👍

**Comment by loreanvictor** [comment]
> Darn it.. yeah that was it.
> 
> Look I am really sorry, it's always funny coming back to some code after a decent break. Must have completely forgotten this detail and then missed it in the docs to just for extra good measure. Thank you 👍
> 
> Perhaps it was added automatically last time as my previous repo's (working deployments) `config.ts` did not have that attribute set. Either way its all good for now.

yeah actually it might still be a bug, the CLI is supposed to automatically fill the namespace whenever it is initializing codedoc for a project that has git info.

and even if it is not, there is absolutely no need to apologize, the least amount of information this conveys is that project namespace can be confusing to people at times and perhaps that requires better docs / measures / etc.

**Comment by beauwilliams** [comment]
That could definitely be the case then, as I have not seen this behaviour in the past. However cautious that I rolled back release and still had some issue, makes me think it could be something on my machine.

Tried again and was definitely not importing the path of namespace though, did the usual `codedoc init` && `codedoc build` --> Verified the issue by looking at `index.html` to see that the root path was `/docs/*` and not `$NAMESPACE/docs/*` as you would expect.

But then.. I realised when testing this morning I forgot to `git init` my test repo (also ran `hub create` too to give it an upstream repo in case it was required). After that and fresh `codedoc init && codedoc build` the namespace was correctly identified and I could see it in the paths in the `index.html`. All good!

I suspect there was an issue with my repo earlier, and then that was compounded perhaps by forgetting to init my test repo so there was no namespace that could be determined.

You are right.. perhaps some docs clarification might help here
